docs/introduction/developer-quickstart.md

```diff
@@ -107,19 +114,18 @@ contract StorageExample {
 }
 ```
 
-### Explanation:
+### Explanation
 
-We've now made our data variable private, meaning it can't be accessed directly from outside the contract. To fetch its value, we've provided a custom public view function `getValue` where the user provides the address. On both Ethereum and Ten, if you call this function you will retrieve the number stored by that address.
+The `storedValues` variable is now private, and we added a basic `getValue` function for users to retreive their value. 
 
-:::caution
-In Ethereum, the `_storedValues` variable can also be accessed directly using the `getStorageAt` method, but not in Ten.
-:::
+On both Ethereum and TEN, anyone can call `getValue` and retrieve any value.   
+On Ethereum, `_storedValues` can also be accessed directly with `getStorageAt`
 
-## Step 3: Implementing Data Access Control 
+## Step 3: Adding Data Access Control
```

Comment:
I think we should use the same verbiage as in the first "Key migration steps" section so either title this "Data Protection Logic" or change that to "Data Access Control" just so the eye makes the immediate link